pulpcore/pulpcore/app/viewsets/base.py

```diff
@@ -14,6 +14,16 @@
 from rest_framework.generics import get_object_or_404
 from rest_framework.schemas import AutoSchema
 
+# These should be used to prevent duplication and keep things consistent
+NAME_FILTER_OPTIONS = ['exact', 'in']
+# e.g.
+# /?name=foo
+# /?name__in=foo,bar
+DATETIME_FILTER_OPTIONS = ['lt', 'lte', 'gt', 'gte', 'range']
+# e.g.
+# /?created__gte=2018-04-12T19:45:52
```

Comment:
When I tried these filters forever ago, I also found them hard to use. This comment is helpful, but it is still unclear what the rules are.